webapp/src/components/chat/shared/Alerts.tsx

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import {
+    makeStyles,
+    tokens
+} from '@fluentui/react-components';
+import { Alert } from '@fluentui/react-components/unstable';
+import { Dismiss16Regular } from '@fluentui/react-icons';
+import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../../redux/app/hooks';
+import { RootState } from '../../../redux/app/store';
+import { removeAlert } from '../../../redux/features/app/appSlice';
+
+const useClasses = makeStyles({
+    root: {
+    },
+    alert: {
+        fontWeight: tokens.fontWeightRegular,
+        color: tokens.colorNeutralForeground1,
+        backgroundColor: tokens.colorNeutralBackground6,
+        fontSize: tokens.fontSizeBase200,
+        lineHeight: tokens.lineHeightBase200,
+    },
+});
+
+export const Alerts: React.FC = () => {
```

Comment:
We already have an Alerts component under components/shared

Reply:
Removed